src/components/card.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'React';
+
+type CardProps = {
+    title: string,
+    url: string
+};
+
+const Card = ({ title, url }: CardProps) => {
+    return (
+        <>
```

Comment:
Can you run `npm run fmt` so that everything you've added is formatted? We don't have a Github action to do this automatically like we do in the PagesJS repo.